### PR TITLE
Sonata press article change

### PR DESCRIPTION
**Pull request by ngdo-pro**
Hey guys 

Remember to : 

+ Composer update
+ Composer install
+ php app/console doctrine:generate:entities BookEditorBundle
+ php app/console doctrine:schema:update --force
+ php app/console doctrine:fixture:load

known issue :
You can't edit press articles from Book, you have to edit them in press articles

### CODE REVIEW

File: src/BookEditorBundle/Entity/Book.php [hunk 1]
```diff
@@ -42,21 +44,39 @@ class Book
...
+    private $slug;
 
     /**
      * @var \DateTime
      */
-    private $releaseDate;
+    private $uploaded;
 
     /**
+<<<<<<< HEAD
```
Comment (Nagx) [inline src/BookEditorBundle/Entity/Book.php]: it's normaly ?

Comment (ngdo-pro) [inline src/BookEditorBundle/Entity/Book.php]: adressed, thx !

**Comment by christianJCK** [comment]
There is a problem with articles, when I create one it goes to every book.
:-1:

**Comment by ngdo-pro** [comment]
i just adressed this issue

**Pull request merged**